src/templates/React InstantSearch widget/src/lib/widget.tsx.hbs

```diff
@@ -1,14 +1,15 @@
 import { {{ pascalCaseName }}Component } from './component';
 import { connect{{ pascalCaseName }} } from './connector';
 
-import type { ElementType } from 'react';
-
 type WidgetParams = {
   /**
    * Placeholder text for input element.
    */
   placeholder?: string;
 };
 
-export const {{ pascalCaseName }}: ElementType<WidgetParams> =
-  connect{{ pascalCaseName }}({{ pascalCaseName }}Component);
+export const {{ pascalCaseName }}: React.ElementType<WidgetParams> = connect{{ pascalCaseName }}(
```

Comment:
The change from `ElementType` to `React.ElementType` fixes a TypeScript error during `build:types`. I don't know why though.